docs/platform-support.rst

```diff
@@ -215,3 +215,33 @@ NVIDIA Network Operator has been validated in the following scenarios:
      - Yes
      - 
 
+
+=============================================
+Supported Precompiled Images for DOCA Drivers
+=============================================
+
+--------
+Overview
+--------
+
+To save startup (compilation) time and operational effort, precompiled DOCA driver images are available for common OS/flavor/kernel variants.
+
+The container image tag pattern used for common variants is: **driver_ver-container_ver-kernel_ver-flavor-os**. For example: ``24.07-0.6.1.0-6.8.0-49-generic-ubuntu24.04``
+
+**NOTE:** The tags are multi-architecture manifests, so no need to state the architecture.
+
+**NOTE:** For the *generic* flavor of Ubuntu, the default Kernel version is used for precompiling (e.g. *6.8.0-31* for Ubuntu 24.04). Whereas for all other flavors, their latest (at time of DOCA packaging/release) Kernel version is used.
+
+----------------------------
+Limitations and Restrictions
+----------------------------
+
+- Support for deploying the driver containers with precompiled drivers is currently limited to hosts with Ubuntu 22.04 or 24.04 (both amd64 and arm64).
+- NVIDIA supports precompiled driver containers for the most recently released DOCA-OFED GA drivers.
+- NVIDIA builds precompiled driver containers for generic, nvidia, aws, azure, and oracle kernel variants.
```

Comment:
'nvidia, aws, azure, and oracle' => 'NVIDIA, AWS, Azure, and Oracle'

Reply:
and 'generic' should remain lowercase?

Reply:
I suggest to have quotes around the variant names and keep the same as shown in the CLI.
`generic`, `nvidia`, `aws` ....